### DISCUSSION TITLE
Not a request

### DISCUSSION THREAD

**Discussion by JohannesBaader** [mworion/MountWizzard4#350] [Feature Request]
Hello Community, 

just seen your future actionlist to add Baader Domes to the mountwizzard4 . Will be happy to help in this, if you should reach that point, let me know. 

I will be starting with MountWizzard on a AZ5000 out of curiosity ;-). I will report our findings and hope for help incase we get stucked. 

Wishing everybody a lovely weekend! 

clear skies,

Johannes

**Comment by Ruediger3** [comment]
Hi Michel,

I am in touch with Johannes for getting MW4 up and running. In case we get stuck, I will ping you. 
For the Dome topic I am out of the race, since I do not have one. 

Cheers
Rüdiger

**Comment by mworion** [comment]
Hi Rüdiger, Johannes,
I'm happy to help. MW4 should run with an AZ5000 (still did not test is as I have an GM1000). Only topic which is not actually in is the 3D part and I only had modeled the GM1000-4000.
Regarding the dome Johannes: if all Baader Domes are connected via the RS-232 to the mount and directed through the mount, I should have a first hand spec out of the documentation.
Michel

**Comment by JohannesBaader** [comment]
Hi Michael,

No need for fancy 3D stuff. As long as we can get it working. Had lots of problems with other software. Not shure if this is because our customer is using a FLI Kepler Camera. It seems to have issues and problems all the time. Any change in software because of the fact the mount is not equatorial?

Hardly the dome connection on 10 micron mounts has been ever used in the last 5 years or so. Really all our installations would have the dome and mount connected to a PC. Not interconnected. Our dome’s had RS232 interface only till this year. From now on we run on Rj45 plugs and TCPIP. We still have RJ232 as a option but not used much. Using ASCOM Alpaca is really great and the driver just works well. We can even pull encoder readouts from the mount via network in our controler an just follow the mount by that.

Never the less just wanted to let you know, I am more the open to share our protocol in case you want it. Can send you the details next week.

Best regards,
Johannes Baader         [https://assets.baader-planetarium.com/media/extendware/ewimageopt/media/skin/41/a/logo.png]  <https://www.baader-planetarium.com>
CEO
Baader Planetarium GmbH
Zur Sternwarte 4 • DE-82291 Mammendorf
Tel.: +49 8145 8089-0
[https://www.baader-planetarium.com/media/tracking/logo_youtube_email.png] <https://www.youtube.com/baaderplanetarium>  [https://www.baader-planetarium.com/media/tracking/logo_facebook_email.png]  <https://www.facebook.com/baaderplanetarium>       [https://www.baader-planetarium.com/media/tracking/logo_twitter_email.png]  <https://www.twitter.com/baader_astro>      [https://www.baader-planetarium.com/media/tracking/logo_instagram_email.png]  <https://www.instagram.com/baaderplanetarium>     [https://www.baader-planetarium.com/media/tracking/logo_linkedin_email.png]  <https://www.linkedin.com/company/baader-planetarium>

Please note our information on confidentiality<https://www.baader-planetarium.com/en/confidentiality/> and data protection<https://www.baader-planetarium.com/en/privacy-statement/>. We reserve the rights of errors
Amtsgericht München, HRB 99721, VAT-ID: DE152112800 • Geschäftsführer: T. Baader, J. Baader
________________________________
Von: Michael ***@***.***>
Gesendet: Friday, March 8, 2024 8:21:28 PM
An: mworion/MountWizzard4 ***@***.***>
Cc: Johannes Baader ***@***.***>; Author ***@***.***>
Betreff: Re: [mworion/MountWizzard4] Not a request (Discussion #350)


Hi Rüdiger, Johannes,
I'm happy to help. MW4 should run with an AZ5000 (still did not test is as I have an GM1000). Only topic which is not actually in is the 3D part and I only had modeled the GM1000-4000.
Regarding the dome Johannes: if all Baader Domes are connected via the RS-232 to the mount and directed through the mount, I should have a first hand spec out of the documentation.
Michel

**Reply by mworion** [reply]
Hi Johannes,
for MW4 I did not rely on any change of an non equatorial mount. There were some special command in the protocol spec, but I did not use them at all. 
For the dome interface: If you are anyway support an ASCOM driver or even better an ALPACA driver, the dome function should work out of the box. Still I need to check the drawings for the non equatorial mounts (I gathered the GMxxxx from 10 micron website in bad quality), because the advanced dome functions (minimize dome movement etc) need a precise geometry. If need I will add these types in the MW4 database as well. 
At least as Rüdiger said: if you have any issues with MW4, please let me know. I'm happy to help. Also for some new desired functions as well. 
Michel

**Comment by JohannesBaader** [comment]
Hi Michel,
Hi Rüdiger

The Baader Domes latest controler run under Alpaka. The older ones connected via RS233 not really. Only normal ASCOM. We can wait for a reale use case and make it happen then.

Regarding the MW for AZ I will inform you and Rüdiger if the customer has time for me to test it. Thanks for your straight forward approach.

I am out of the company till the end of the week so but much will happen before that.

Best regards,

Johannes
Johannes Baader         [https://assets.baader-planetarium.com/media/extendware/ewimageopt/media/skin/41/a/logo.png]  <https://www.baader-planetarium.com>
CEO
Baader Planetarium GmbH
Zur Sternwarte 4 • DE-82291 Mammendorf
Tel.: +49 8145 8089-0
[https://www.baader-planetarium.com/media/tracking/logo_youtube_email.png] <https://www.youtube.com/baaderplanetarium>  [https://www.baader-planetarium.com/media/tracking/logo_facebook_email.png]  <https://www.facebook.com/baaderplanetarium>       [https://www.baader-planetarium.com/media/tracking/logo_twitter_email.png]  <https://www.twitter.com/baader_astro>      [https://www.baader-planetarium.com/media/tracking/logo_instagram_email.png]  <https://www.instagram.com/baaderplanetarium>     [https://www.baader-planetarium.com/media/tracking/logo_linkedin_email.png]  <https://www.linkedin.com/company/baader-planetarium>

Please note our information on confidentiality<https://www.baader-planetarium.com/en/confidentiality/> and data protection<https://www.baader-planetarium.com/en/privacy-statement/>. We reserve the rights of errors
Amtsgericht München, HRB 99721, VAT-ID: DE152112800 • Geschäftsführer: T. Baader, J. Baader
________________________________
Von: Michael ***@***.***>
Gesendet: Samstag, März 9, 2024 15:27
An: mworion/MountWizzard4 ***@***.***>
Cc: Johannes Baader ***@***.***>; Author ***@***.***>
Betreff: Re: [mworion/MountWizzard4] Not a request (Discussion #350)


Hi Johannes,
for MW4 I did not rely on any change of an non equatorial mount. There were some special command in the protocol spec, but I did not use them at all.
For the dome interface: If you are anyway support an ASCOM driver or even better an ALPACA driver, the dome function should work out of the box. Still I need to check the drawings for the non equatorial mounts (I gathered the GMxxxx from 10 micron website in bad quality), because the advanced dome functions (minimize dome movement etc) need a precise geometry. If need I will add these types in the MW4 database as well.
At least as Rüdiger said: if you have any issues with MW4, please let me know. I'm happy to help. Also for some new desired functions as well.
Michel

**Comment by JohannesBaader** [comment]
Hi all,

meanwhile we spend more time with this problem and found that the ultimate culprit for the issue was the FLI Camera that was delivering crab data that we could not process with Plate Solve. This is solved now and we could reach a solution for the moment.

Never the less, MW4 is on our todo list and we still want to get familiar with it. Just the urgency has vanished to some extend.

We will get in touch with you if we find the time to make a new session regarding this.

THANKS for your willingness to help. This by its own, drives us in MW4 direction for future usage. Thanks!

Johannes Baader
[baader-planetarium.com]<https://www.baader-planetarium.com/>
CEO
Baader Planetarium GmbH
Zur Sternwarte 4 • DE-82291 Mammendorf
Tel.: +49 8145 8089-0
[YouTube]<https://www.youtube.com/baaderplanetarium>
[Facebook]<https://www.facebook.com/baaderplanetarium>
[Twitter]<https://www.twitter.com/baader_astro>
[Instagram]<https://www.instagram.com/baaderplanetarium>
[LinkedIn]<https://www.linkedin.com/company/baader-planetarium>


Please note our information on confidentiality<https://www.baader-planetarium.com/en/confidentiality/> and data protection<https://www.baader-planetarium.com/en/privacy-statement/>. We reserve the rights of errors
Amtsgericht München, HRB 99721, VAT-ID: DE152112800 • Geschäftsführer: T. Baader, J. Baader


Von: Michael ***@***.***>
Gesendet: Samstag, 9. März 2024 15:27
An: mworion/MountWizzard4 ***@***.***>
Cc: Johannes Baader ***@***.***>; Author ***@***.***>
Betreff: Re: [mworion/MountWizzard4] Not a request (Discussion #350)


Hi Johannes,
for MW4 I did not rely on any change of an non equatorial mount. There were some special command in the protocol spec, but I did not use them at all.
For the dome interface: If you are anyway support an ASCOM driver or even better an ALPACA driver, the dome function should work out of the box. Still I need to check the drawings for the non equatorial mounts (I gathered the GMxxxx from 10 micron website in bad quality), because the advanced dome functions (minimize dome movement etc) need a precise geometry. If need I will add these types in the MW4 database as well.
At least as Rüdiger said: if you have any issues with MW4, please let me know. I'm happy to help. Also for some new desired functions as well.
Michel

**Comment by mworion** [comment]
Johannes,
Now as the urgency is gone, we could plan a meeting for MW4 (willing to visit you at your site as I anyhow intended privately to show up (as as customer).
Please let me know, when a good time is about and I look what my job allows.
Michel

**Comment by JohannesBaader** [comment]
Hi Michael,


Thats a great idea! We can do that around July maybe. But I need to check with the assembly team. They are traveling allot now and we will hardly find a time when all related people are here. Never the less. Better with some of the guys then never…

So if we do it, mid of July maybe.

Best regards,

Johannes

Johannes Baader         [https://assets.baader-planetarium.com/media/extendware/ewimageopt/media/skin/41/a/logo.png]  <https://www.baader-planetarium.com>
CEO
Baader Planetarium GmbH
Zur Sternwarte 4 • DE-82291 Mammendorf
Tel.: +49 8145 8089-0
[https://www.baader-planetarium.com/media/tracking/logo_youtube_email.png] <https://www.youtube.com/baaderplanetarium>  [https://www.baader-planetarium.com/media/tracking/logo_facebook_email.png]  <https://www.facebook.com/baaderplanetarium>       [https://www.baader-planetarium.com/media/tracking/logo_twitter_email.png]  <https://www.twitter.com/baader_astro>      [https://www.baader-planetarium.com/media/tracking/logo_instagram_email.png]  <https://www.instagram.com/baaderplanetarium>     [https://www.baader-planetarium.com/media/tracking/logo_linkedin_email.png]  <https://www.linkedin.com/company/baader-planetarium>

Please note our information on confidentiality<https://www.baader-planetarium.com/en/confidentiality/> and data protection<https://www.baader-planetarium.com/en/privacy-statement/>. We reserve the rights of errors
Amtsgericht München, HRB 99721, VAT-ID: DE152112800 • Geschäftsführer: T. Baader, J. Baader
Von: Michael ***@***.***>
Gesendet: Freitag, 15. März 2024 17:57
An: mworion/MountWizzard4 ***@***.***>
Cc: Johannes Baader ***@***.***>; Author ***@***.***>
Betreff: Re: [mworion/MountWizzard4] Not a request (Discussion #350)

Johannes,
Now as the urgency is gone, we could plan a meeting for MW4 (willing to visit you at your site as I anyhow intended privately to show up (as as customer).
Please let me know, when a good time is about and I look what my job allows.
Michel

**Comment by mworion** [comment]
Hi Johannes,

That sound like a plan. If you have any ideas about a more defined schedule, please let me know that I could plan hopefully accordingly.

Michel
